src/core/constants/consent-state.js

```diff
@@ -2,26 +2,26 @@
 
 /**
  * Possible consent policy state to proceed with.
- * @enum {number}
+ * @const {!Object<string, number>}
  */
 export const CONSENT_POLICY_STATE = {
   // Enum value has external dependency. Please do not change existing value.
   // If new values are added, please notify the AMP for Ads team to assure
   // correct Real Time Config behavior is maintained for Fast Fetch.
-  SUFFICIENT: 1,
-  INSUFFICIENT: 2,
-  UNKNOWN_NOT_REQUIRED: 3,
-  UNKNOWN: 4,
+  'SUFFICIENT': 1,
+  'INSUFFICIENT': 2,
+  'UNKNOWN_NOT_REQUIRED': 3,
+  'UNKNOWN': 4,
 };
 
 /**
  * Defines valid consent string types passed
  * in by consent iframe metadata.
  * @enum {number}
  */
-export const CONSENT_STRING_TYPE = {
+export const CONSENT_STRING_TYPE_ENUM = {
```

Comment:
was CONSENT_STRING_TYPE_ENUM already passing the required constraints? seems like CONSENT_POLICY_STATE was the only violation.

Reply:
Yes.